devtools/cli/src/main/java/io/quarkus/cli/QuarkusCli.java

```diff
@@ -3,32 +3,26 @@
 import java.io.PrintWriter;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.LinkedList;
-import java.util.concurrent.Callable;
 
 import javax.inject.Inject;
 
-import io.quarkus.cli.core.BuildsystemCommand;
-import io.quarkus.cli.core.ExecuteUtil;
+import io.quarkus.cli.core.BuildSystemRunner;
 import io.quarkus.cli.core.QuarkusCliVersion;
 import io.quarkus.devtools.project.BuildTool;
 import io.quarkus.devtools.project.QuarkusProject;
 import io.quarkus.runtime.QuarkusApplication;
 import io.quarkus.runtime.annotations.QuarkusMain;
+import picocli.AutoComplete;
 import picocli.CommandLine;
 
 @QuarkusMain
-@CommandLine.Command(name = "qs", versionProvider = QuarkusCliVersion.class, subcommandsRepeatable = true, mixinStandardHelpOptions = true, subcommands = {
-        Build.class,
-        Clean.class, Create.class, CreateJBang.class, List.class, Platforms.class, Add.class, Remove.class, Dev.class,
-        CreateExtension.class })
+@CommandLine.Command(name = "quarkus", versionProvider = QuarkusCliVersion.class, subcommandsRepeatable = true, mixinStandardHelpOptions = true, subcommands = {
+        Create.class, CreateCli.class, CreateExtension.class,
+        Build.class, Clean.class, Dev.class,
+        List.class, Add.class, Remove.class,
```

Comment:
Same here about `Add` and `Remove`, it's definitely not clear enough. You need to know what you are adding or removing. Especially since you have `CreateExtension`.

Reply:
Apart from clarity, what's also bothering me is that we paint ourselves into a corner in case we want to `add` other things later. And that doesn't seem that far fetched. Same for `list`. It's very generic and I'm pretty sure we will want to list other things at some point.

I would rather have `ListExtensions`, `AddExtension` and `RemoveExtension`.

Reply:
The internal class names are for reference only (and I tried to group them in the list of subcommands also, to show they are related).

Reply:
I like the idea of clear and consistent names, i.e. `qs list-extensions` and `ListExtensions`